docs/install.md

```diff
@@ -3,35 +3,48 @@
 This documents any required information either during installation or
 post installation to ensure the operator can function correctly.
 
-## STS Clusters
-
-### Post operator installation
+- [STS Clusters](#sts-clusters)
+    - [Post operator installation](#post-operator-installation)
+        - [Use CredentialsRequest created by the operator](#use-credentialsrequest-created-by-the-operator)
+        - [Use predefined secret](#use-predefined-secret)
+            - [Option 1. Using ccoctl](#option-1-using-ccoctl)
+            - [Option 2. Using AWS CLI](#option-2-using-aws-cli)
 
-In an STS Cluster, `CredentialsRequest`s are not automatically provisioned by
-the **cloud-credential-operator** and the manual intervention done by the
-cluster-admin is required. IAM role and policies as well as the credentials secret need to be provisioned manually for the further consumption by the pods.
-`ccoctl` binary can be used to facilitate this task.
+## STS Clusters
+In an STS cluster, **cloud-credential-operator** do not automatically provision secrets for `CredentialsRequest`s
+and the manual intervention done by the cluster-admin is required.
+IAM role and policies as well as the credentials secret need to be provisioned manually for the further consumption by the pods.
 
 Normally, the **aws-load-balancer-operator** relies on the **cloud-credential-operator**
-to provision the secret for the operated controller using `CredentialsRequest`. And so in an STS cluster this
+to provision the secret for the operated controller using `CredentialsRequest`. However, in an STS cluster this
 secret needs to be provisioned manually. The **aws-load-balancer-operator** will wait until the required
 secret is created and available before spawning the **aws-load-balancer-controller** pod.
 
-#### Pre-Requisites
+### Post operator installation
+
+There are three ways to manually create the required IAM resources:
+- Using a `CredentialsRequest` generated by the operator for an existing `AWSLoadBalancerController`.
+- Using a pre-defined `CredentialsRequest` from this repository for an `AWSLoadBalancerController` that has not yet been created.
+- Employing AWS CLI commands for an `AWSLoadBalancerController` that has not yet been created.
```

Comment:
I'm not sure what happened here, but this doesn't match the titles of the three ways you listed below, which are:

1. Use CredentialsRequest created by the operator
2. Use a Predefined Secret and ccoctl
3. Use a Predefined Secret  and the AWS CLI

Reply:
I tried to make it better phrased to match the titles.
My idea was to keep it focused on the functional aspect - creation of the IAM resources making the choice of the concrete client (`ccoctl` or `aws`) a technical detail to be described down the document.